Localization/de-DE.hjson

```diff
@@ -2,30 +2,30 @@ Mods: {
 	BrighterLight: {
 		Configs: {
 			Config: {
-				// DisplayName: Server/Singleplayer Config
+				DisplayName: Server/Singleplayer Konfiguration
 
 				Brightness: {
-					// Label: Brightness
-					// Tooltip: Brightness Multiplier
+					Label: Helligkeit
+					Tooltip: Helligkeitsmultiplikator
 				}
 			}
 
 			MPClientConfig: {
-				// DisplayName: Multiplayer Client Config
-				// Headers.MultiplayerOnly: Multiplayer Only: Change your own brightness here
+				DisplayName: Multiplayer Client Konfiguration
+				Headers.MultiplayerOnly: Nur Multiplayer: Ändere hier deine eigene Helligkeit
 
 				Brightness: {
-					// Label: "{$Configs.Config.Brightness.Label}"
-					// Tooltip: "{$Configs.Config.Brightness.Tooltip}"
+					Label: "{$Configs.Config.Brightness.Label}"
+					Tooltip: "{$Configs.Config.Brightness.Tooltip}"
 				}
 
 				UseServerSettings: {
-					// Label: Use Server Settings
-					/* Tooltip:
+					Label: Verwende Server Einstellungen
+					Tooltip:
+						'''
+						Wenn aktiviert, überschreibt '{$Configs.Config.Brightness.Label}' vom Server (via '{$Configs.Config.DisplayName}') diese Einstellung
```

Comment:
`Wenn aktiviert, wird '{$Configs.Config.Brightness.Label}' vom Server (via '{$Configs.Config.DisplayName}') überschrieben`

Reply:
Yes, that sounds better.👍